trunk/SUAVE/Analyses/Aerodynamics/AVL.py

```diff
@@ -1,152 +1,88 @@
+# Default aero Results
+from Results import Results
 
+# The aero methods
+from SUAVE.Methods.Aerodynamics import Fidelity_Zero as Methods
+from Process_Geometry import Process_Geometry
+from SUAVE.Analyses.Aerodynamics.AVL_Inviscid import AVL_Inviscid
 
 # ----------------------------------------------------------------------
-#  Class
+#  Analysis
 # ----------------------------------------------------------------------
-
-class AVL(Aero_Analysis):
-    """ SUAVE.Analyses.Aerodynamics.AVL
-        aerodynamic model that performs a vortex lattice analysis using AVL
-        (Athena Vortex Lattice, by Mark Drela of MIT).
-
-        this class is callable, see self.__call__
-
-    """
-
+class AVL(Markup):
+    
     def __defaults__(self):
-        self.tag        = 'avl'
-        self.keep_files = True
-
-        self.settings = Settings()
-
-        self.current_status = Data()
-        self.current_status.batch_index = 0
-        self.current_status.batch_file  = None
-        self.current_status.deck_file   = None
-        self.current_status.cases       = None
         
-        self.features = None
-
-
-    def finalize(self):
-
-        features = self.features
-        self.tag      = 'avl_analysis_of_{}'.format(features.tag)
-
-        run_folder = self.settings.filenames.run_folder
-        if os.path.exists(run_folder):
-            if self.keep_files:
-                warn('deleting old avl run files',Warning)
-            rmtree(run_folder)
-        os.mkdir(run_folder)
-
-        return
-
-
-    def evaluate(self,state,**args):
-        
-        # unpack
-        conditions = state.conditions
-        results = self.evaluate_conditions(conditions)
+        self.tag    = 'AVL_markup'       
+    
+        # Correction factors
+        settings = self.settings
+        settings.trim_drag_correction_factor        = 1.02
+        settings.wing_parasite_drag_form_factor     = 1.1
+        settings.fuselage_parasite_drag_form_factor = 2.3
+        settings.oswald_efficiency_factor           = None
+        settings.viscous_lift_dependent_drag_factor = 0.38
+        settings.drag_coefficient_increment         = 0.0000
+        settings.spoiler_drag_increment             = 0.00 
+        settings.maximum_lift_coefficient           = np.inf 
+        settings.half_mesh_flag                     = True
+        settings.parallel                           = False
+        settings.processors                         = 1
+        settings.vsp_mesh_growth_ratio              = 1.3
+        settings.vsp_mesh_growth_limiting_flag      = False
+                
+        # Build the evaluation process
+        compute = self.process.compute
+        compute.lift = Process()
+
+        # Run SU2 to determine lift
+        compute.lift.inviscid                         = AVL_Inviscid()
+        compute.lift.total                            = SUAVE.Methods.Aerodynamics.AERODAS.AERODAS_setup.lift_total
```

Comment:
@timdmacdo: we may need to have a common repository. This function he's referring to is analysis agnostic. I think we should have common scripts as much as possible to reduce redundant code.